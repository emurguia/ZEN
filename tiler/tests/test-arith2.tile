// Testing arithmetic operations

init{
	int a;
	int b;
	int c;
	tile(3,3);

	a = 1;
	b = 2;
	c = 3;
	iprint(a+b/c);
	iprint(b-c);
	iprint(a*c);
	iprint(c*c*c*c*c);
	iprint(b%c);
	close();
}