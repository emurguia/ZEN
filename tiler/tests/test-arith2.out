1
-1
3
243
2
The game window was successfully launched ...
The game window has been exited. Program exited.
Game over.